packages/suite/src/views/wallet/anonymize/components/CoinjoinConfirmation.tsx

```diff
@@ -2,7 +2,6 @@ import React, { useState } from 'react';
 import { useDispatch, useSelector } from '@suite-hooks';
 import styled from 'styled-components';
 
-import { RECOMMENDED_SKIP_ROUNDS } from '@suite/services/coinjoin';
```

Comment:
it's not used anymore, right? replaced by SKIP_ROUNDS_VALUE_WHEN_ENABLED, so it can be deleted from coinjoin/config.ts

Reply:
Right, a851f8c3294615964c77300bfd950a105eb10121.